docs/.vuepress/plugins/sidebar/Sidebar.vue

```diff
@@ -1,27 +1,25 @@
 <template>
-  <SidebarItem :pages="pages" :item="item" />
+  <div>hi</div>
+  <div v-for="route in htmlRoutes" :key="route.path">
+    <p @click="router.push(route.path)">{{ route.meta.title }}</p>
+  </div>
 </template>
 
 <script>
-import SidebarItem from "./SidebarItem";
-
 export default {
-  name: "sidebar",
-  components: {
-    SidebarItem,
-  },
+  name: 'sidebar',
   props: {
-    item: Object,
-    pages: Array,
+    router: Object,
   },
-  data() {
-    return {};
+  setup(props) {
+    const htmlRoutes = props.router
+      .getRoutes()
+      .filter(route => /html$/.test(route.path));
+    return {
+      htmlRoutes,
+    };
   },
-  computed: {},
-  created() {},
-  destroyed() {},
-  methods: {},
 };
 </script>
 
-<style lang="stylus" scoped></style>
+<style></style>
```

Comment:
If we don't plan any styles here it could be removed.

Reply:
As I mentioned in the commit message, the old styles made in stylus need to be rewritten to scss. In my opinion, it's better to have them in the project and base on them than write everything once again.

Yes, the style tag can be removed for now but we'll probably style the sidebar later.